__tests__/app.test.js

```diff
@@ -168,4 +167,65 @@ describe("GET /api/articles?sort_by=votes / article_id/created_at/comment_count"
             expect(body.msg).toBe("Bad Request invalid sort_by")
         })
     })
+})
+
+describe("POST /api/articles/:article_id/comments",()=>{
+
+    test("201 /api/articles/:article_id/comments should add a comment about a specific article as long as properties are correct",()=>{
+
+        const newComment = {
+            username: "rogersop",
+            body: "wow this article shows the REAL truth that THE MAN is trying to hide from us"
+        }
+        return request(app).post("/api/articles/4/comments").send(newComment).expect(201)
+        .then(({body})=>{
+
+            expect(body[0].comment_id).toBe(19)
+            expect(body[0].author).toBe("rogersop")
+            expect(body[0].body).toBe('wow this article shows the REAL truth that THE MAN is trying to hide from us')
+            expect(body[0].article_id).toBe(4)
+        })
+    })
+
+    test("404 /api/articles/:article_id/comments with an invalid article_id should give a 404 Not Found",()=>{
```

Comment:
This description is not quite right as the id is valid just not found - 404 is appropriate though